test

```diff
@@ -52,13 +52,8 @@ if [ $MACHINE_TYPE != "armv7l" ]; then
   RACE="--race"
 fi
 
-go test -timeout 3m ${COVER} $@ ${TEST} ${RACE} -cpu 1,2,4
-go test -timeout 3m ${COVER} $@ ${NO_RACE_TEST} -cpu 1,2,4
-
-if [ -n "$INTEGRATION" ]; then
```

Comment:
do not remove this. Travis will not trigger integration test since it does not set $INTEGRATION env in travis.yml.